mapboxcli/scripts/geocoding.py

```diff
@@ -98,7 +101,7 @@ def geocoding(ctx, query, forward, include_headers, lat, lon,
         for q in iter_query(query):
             try:
                 resp = geocoder.forward(
-                    q, types=place_type, lat=lat, lon=lon, country=country)
+                    q, types=place_type, lat=lat, lon=lon, country=country, bbox=bbox)
```

Comment:
@perrygeo @mattficke let's match [fio-cat](https://github.com/Toblerity/Fiona/blob/master/fiona/fio/cat.py#L29). In other words, both `--bbox "w,s,e,n"` (comma separated) and `--bbox "[w,s,e,n]"`.